packages/backend/src/services/SavedChartsService/SavedChartService.ts

```diff
@@ -72,6 +81,16 @@ export class SavedChartService {
                 savedChart.metricQuery.tableCalculations.length,
             pivotCount: (savedChart.pivotConfig?.columns || []).length,
             chartType: savedChart.chartConfig.type,
+            constantReferenceLinesNum: echartsConfig?.series?.filter(
```

Comment:
I know Katie suggested these properties names but I would change them to match the existing ones. 
`referenceLinesCount` and `conditionalFormattingRulesCount`
What do you think ?

Reply:
We also have a section for the cartesian metadata in line 94. Should we move that one in there ? 
and have a new one for the table metadata?

Reply:
yep, naming is great! consistency is our friend ❤️

Reply:
Updated 
![Screenshot from 2023-01-30 12-14-56](https://user-images.githubusercontent.com/1983672/215464141-e4b58a89-a2e1-4064-b0b1-0ae6692afb58.png)